code/style.css

```diff
@@ -1,4 +1,314 @@
-html {
-  background: #0025ff;
+* {
+ padding: 0px; 
+ margin: 0px;
+
+}
+
+
+/* Header with logo and nav */
+header {
+  display:flex;
+  width: 100%;
+  justify-content: space-between;
+  z-index: 1;
+
+  background-color: #fff;
+}
+
+.logoIcon  {
+  width: 140px;
+  margin-left: 30px;
+  margin-right: 50px;
+  margin-top: 20px;
+  margin-bottom: 30px;
+}
+
+nav {
+  display: flex;
+  width: 100%;
+  justify-content: space-evenly;
+  align-self: center;
+
+  font-family: 'Gill Sans';
```

Comment:
It's always good to add a second font-family in case a browser can't show this specific one. You can always add sans-serif to be on the safe side.
```
font-family: 'Gill Sans', sans-serif;
```